.eslintignore

```diff
@@ -0,0 +1,5 @@
+content
+docs
+dist
+scripts
```

Comment:
When I lint the scripts, I get

```
✖ 30156 problems (30156 errors, 0 warnings)
```

They are also unlikely to be changed often by new contributors, so linting them is a bit of a waste of time I think.